chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomsReceivedIntegrationTest.kt

```diff
@@ -90,11 +94,15 @@ internal object JoinedRoomsReceivedIntegrationTest : Spek({
                 userSubscriptionDispatcher.onEvent(event.asElementsEvent())
             }
 
-            it("then no actions are dispatched") {
+            it("then no user subscription actions are dispatched") {
```

Comment:
```suggestion
            it("then no joined rooms diff actions are dispatched") {
```
`CurrentUserReceived` is also a "user subscription" related action

Reply:
Hm. I've written that `no joined rooms differ actions` because we call it a differ.

Reply:
actually some of those checked are never produced by the differ (but yeah worth checking) so I'd drop the "differ" word here altogether

Reply:
Hm. Yeah, that sounds fair.